bin/reth/src/config.rs

```diff
@@ -78,3 +80,12 @@ impl Default for SenderRecoveryConfig {
         Self { commit_threshold: 5_000, batch_size: 1000 }
     }
 }
+
+/// Configuration for each stage in the pipeline.
+#[derive(Debug, Clone, Default, Deserialize, Serialize)]
+pub struct DiscoveryConfig {
```

Comment:
hmm, I'm not sure if `Discovery` is semantically fitting here, because these are already "discovered" and will be added to the Peerset directly.

Maybe `Nodes|Peers`Config? I guess `Peers` makes sense, for example there's an rpc endpoint called `admin_addTrustedPeer` which accepts a NodeRecord as string.